bees/twitterbee/twitterbeefactory.go

```diff
@@ -168,6 +168,11 @@ func (factory *TwitterBeeFactory) Events() []bees.EventDescriptor {
 					Description: "text content of the tweet",
 					Type:        "string",
 				},
+				{
+					Name:        "url",
+					Description: "URL of the tweet",
+					Type:        "string",
```

Comment:
The name 'url' is totally fine. It's namespaced to this Bee, anyway, and eventually the type shouldn't be a user-facing value, so I don't mind it being a name and type at the same time.